conda-recipe/run_test.bat

```diff
@@ -28,54 +28,45 @@ if "%PYTHON%"=="python" (
     set NO_DIST=1
 )
 
+
+
 %PYTHON% -c "from sklearnex import patch_sklearn; patch_sklearn()" || set exitcode=1
 
+set "PYTEST_ARGS= "
+
+IF DEFINED COVERAGE_RCFILE (set PYTEST_ARGS=--cov=onedal --cov=sklearnex --cov-config="%COVERAGE_RCFILE%" --cov-append --cov-report= %PYTEST_ARGS%)
+
 rem Note: execute with argument --json-report as second argument
 rem in order to produce a JSON report under folder '.pytest_reports'.
-set with_json_report=0
 if "%~2"=="--json-report" (
-    set with_json_report=1
+    set PYTEST_ARGS=--json-report --json-report-file=.pytest_reports\FILENAME.json %PYTEST_ARGS%
     mkdir .pytest_reports
     del /q .pytest_reports\*.json
 )
 
 echo "NO_DIST=%NO_DIST%"
```

Comment:
Maybe it could print the `PYTEST_ARGS` template here.

Reply:
Added!